packages/terra-filter-view/src/_FilterFrameUtil.js

```diff
@@ -0,0 +1,70 @@
+class FrameUtil {
+  /**
+   * Returns the dropdown style.
+   * @param {Object} props - The component props.
+   * @param {Object} state - The component state.
+   * @return {Object} - The dropdown style.
+   */
+  static dropdownStyle(props, state) {
+    const { maxHeight, width } = state;
+    return Object.assign({}, Object.assign({}, props).style, { maxHeight, width });
+  }
+
+  /**
+   * Determines the dropdown position.
+   * @param {Object} props - The component props.
+   * @param {Object} state - The component state.
+   * @param {ReactNode} target - The select wrapper.
+   * @param {ReactNode} dropdown - The dropdown.
+   * @return {Object} - The calculated dropdown attributes.
+   */
+  static dropdownPosition(props, state, target, dropdown) {
+    const style = Object.assign({}, props).style || {};
```

Comment:
What is the purpose of doing `Object.assign({}, props)`? I don't think I've seen that done before.

Reply:
if props is undefined and we try to access the style key, it results in the error `cannot access style key of undefined` (or something very similar). An alternative option to this could be:
```
const style = (props || {}).style || {};
```